diff --git a/dist/css-to-rn/parseDeclaration.js b/dist/css-to-rn/parseDeclaration.js
index 59ea66e459dbf6922bcd80a6ac014408ca13c2af..48c184c21a1cc1b730182ba9ac995bb7f06a5965 100644
--- a/dist/css-to-rn/parseDeclaration.js
+++ b/dist/css-to-rn/parseDeclaration.js
@@ -1760,7 +1760,7 @@ function parseAspectRatio(aspectRatio) {
     if (aspectRatio.auto) {
         return "auto";
     }
-    else {
+    else if (aspectRatio.ratio) {
         if (aspectRatio.ratio[0] === aspectRatio.ratio[1]) {
             return 1;
         }
@@ -1768,6 +1768,7 @@ function parseAspectRatio(aspectRatio) {
             return aspectRatio.ratio.join(" / ");
         }
     }
+    return undefined;
 }
 function parseDimension({ unit, value }, options) {
     switch (unit) {
diff --git a/src/css-to-rn/parseDeclaration.ts b/src/css-to-rn/parseDeclaration.ts
index bf095a5b430d24f791b8478f5b513d703bd1a02a..83679e0dc68d30615d69001b890668b03fcc661d 100644
--- a/src/css-to-rn/parseDeclaration.ts
+++ b/src/css-to-rn/parseDeclaration.ts
@@ -2641,13 +2641,14 @@ function parseAspectRatio(
 ): RuntimeValueDescriptor {
   if (aspectRatio.auto) {
     return "auto";
-  } else {
+  } else if (aspectRatio.ratio) {
     if (aspectRatio.ratio[0] === aspectRatio.ratio[1]) {
       return 1;
     } else {
       return aspectRatio.ratio.join(" / ");
     }
   }
+  return undefined;
 }
 
 function parseDimension(
